### ISSUE TITLE
Add plot alt-text

### ISSUE THREAD

**Issue by ndporter** [swcarpentry/r-novice-gapminder#951]
### How could the content be improved?

Add alt-text to all R plots throughout lesson.  Use the `fig.alt` attribute in markdown blocks like in lines 64-67 of `/episodes/08-plot-ggplot2.Rmd`. This step is necessary for compliance with WCAG accessibility standards and required for use of lessons in US federal, state, and local government organizations.

`
{r blank-ggplot, message=FALSE, fig.alt="Blank plot, before adding any mapping aesthetics to ggplot()."}
library("ggplot2")
ggplot(data = gapminder)
`

I have not checked all figures. Some (like above) have alt text. Others, like the next one in that episode, do not. Here is [some advice](https://www.datawrapper.de/academy/how-to-write-good-alternative-descriptions-for-your-data-visualization) on writing alt-text for figures.

**Comment by cgross95** [comment]
Hi @ndporter, thanks for bringing this to our attention and for providing the technical steps to resolve it.

This is very important and should be resolved ASAP.
